src/Annotations/Command/QueueImportCommand.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace eLife\Annotations\Command;
+
+use eLife\ApiSdk\ApiSdk;
+use eLife\Bus\Limit\Limit;
+use eLife\Bus\Queue\InternalSqsMessage;
+use eLife\Bus\Queue\WatchableQueue;
+use eLife\Logging\Monitoring;
+use Iterator;
+use Psr\Log\LoggerInterface;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Throwable;
+
+final class QueueImportCommand extends Command
```

Comment:
this was the candidate for `bus-sdk-php` later, I guess

Reply:
Yes